modules/macos_utils/manifests/clean_appstate2.pp

```diff
@@ -0,0 +1,58 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+define macos_utils::clean_appstate2 (
```

Comment:
Can we come up with a better name for this class?

Reply:
Renamed to `clean_appstate_13_plus`